tools/ci/inria/sanitizers/script

```diff
@@ -110,34 +110,23 @@ ASAN_OPTIONS="detect_leaks=0,use_sigaltstack=0" $run_testsuite
 
 #########################################################################
 
-# Thread sanitizer is not working in OCaml 5: too many alarms that are
-# probably benign races in the GC (#11040).  Turn it off.
+# Run the testsuite with ThreadSanitizer support (--enable-tsan) enabled.
+# Initially intended to detect data races in OCaml programs and C stubs, it has
+# proved effective at also detecting races in the runtime (see #11040).
 
-# echo "======== clang 13, thread sanitizer =========="
+echo "======== clang 13, thread sanitizer =========="
 
-# git clean -q -f -d -x
+git clean -q -f -d -x
 
-# # Select thread sanitizer
-# # Don't optimize too much to get better backtraces of errors
+./configure \
+  --enable-tsan
```

Comment:
What about leaving `CC=clang-13` ?  I feel slightly more confident if we know exactly which C compiler is being tested.

Reply:
Done.